lib/ts/apidefs.h.in

```diff
@@ -105,9 +105,9 @@ extern "C" {
 
  */
 typedef struct {
-  char *plugin_name;
-  char *vendor_name;
-  char *support_email;
+  char const *plugin_name;
+  char const *vendor_name;
+  char const *support_email;
```

Comment:
Everything else in this file uses ``const char *``. It would be better to keep a consistent style.